pallets/ddc-staking/src/lib.rs

```diff
@@ -245,6 +320,27 @@ pub mod pallet {
 	#[pallet::getter(fn current_era)]
 	pub type CurrentEra<T> = StorageValue<_, EraIndex>;
```

Comment:
Shall we delete this type?

Reply:
No, it holds DDC era value. But the doc comment should be updated.